projects/plugins/crm/tests/php/rest-api/v4/class-rest-automation-workflows-controller-test.php

```diff
@@ -311,11 +311,16 @@ public function test_update_workflow_success() {
 			'initial_step' => 'updated_step_2',
 			'steps'        => array(
 				'updated_step_1' => array(
-					'slug'           => Contact_Condition::get_slug(),
-					'next_step_true' => 'updated_step_2',
+					'slug'            => Contact_Condition::get_slug(),
+					'next_step_true'  => 'updated_step_2',
+					'next_step_false' => null,
+					'attributes'      => array(),
 				),
 				'updated_step_2' => array(
-					'slug' => Contact_Condition::get_slug(),
+					'slug'            => Contact_Condition::get_slug(),
+					'next_step_true'  => null,
+					'next_step_false' => null,
+					'attributes'      => array(),
```

Comment:
Updated dummy data so they more accurately represents what would be fetched by `Base_Step::to_array()` which makes direct comparisons in tests much easier.